go/pkg/apis/enricher/framework/java/wildfly_detector.go

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package enricher
+
+import (
+	"context"
+	"encoding/xml"
+	"strings"
+
+	"github.com/redhat-developer/alizer/go/pkg/apis/model"
+	utils "github.com/redhat-developer/alizer/go/pkg/utils"
+)
+
+type WildFlyDetector struct{}
+
+type Wildfly_Standalone_Xml struct {
+	SocketBindingGroup struct {
+		SocketBinding []struct {
+			Name string `xml:"name,attr"`
+			Port string `xml:"port,attr"`
+		} `xml:"socket-binding"`
+	} `xml:"socket-binding-group"`
+}
+
+func (o WildFlyDetector) GetSupportedFrameworks() []string {
+	return []string{"WildFly"}
+}
+
+func (o WildFlyDetector) DoFrameworkDetection(language *model.Language, config string) {
+	if hasFwk, _ := hasFramework(config, "org.wildfly.plugins"); hasFwk {
```

Comment:
org.wildfly.plugins is a common groupId for WildFly bootable JAR and WildFly maven plugin. We would need the artifactId (wildfly-maven-plugin) to differenciate them.

Reply:
Same as above. I think checking both groupId and artifactId will solve this.

Reply:
Done :)